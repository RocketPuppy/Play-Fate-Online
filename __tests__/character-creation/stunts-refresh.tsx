import * as React from "react";
import * as renderer from "react-test-renderer";
import StuntsRefresh from "../../src/character-creation/stunts-refresh";

test("renders all the pieces", () => {
  const render = renderer.create(<StuntsRefresh />);

  expect(render.toJSON()).toMatchSnapshot();
});
