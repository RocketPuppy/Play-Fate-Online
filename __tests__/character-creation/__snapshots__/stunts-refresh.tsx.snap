// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Stunts and Refresh
  </h2>
  <ul>
    <li>
      Three stunts = Refresh of 3
    </li>
    <li>
      Four stunts = Refresh of 2
    </li>
    <li>
      Five stunts = Refresh of 1
    </li>
  </ul>
</div>
`;
